examples/pybullet/gym/pybullet_envs/gym_sanity_check.py

```diff
@@ -0,0 +1,18 @@
+import gym
+import numpy as np
+import pybullet_envs
+envs = [spec.id for spec in gym.envs.registry.all() if spec.id.find('Bullet') >= 0]
+bugged_envs = []
+for env_name in envs:
+    try:
+        print('[TESTING] ENV', env_name, '...')
+        env = gym.make(env_name)
+        env.reset()
+        env.step(np.random.random(env.action_space.shape))
```

Comment:
Better to use env.action_space.sample()
See also https://github.com/bulletphysics/bullet3/blob/master/examples/pybullet/gym/pybullet_envs/examples/testEnv.py
~~~~~~~~~~~~
sample = env.action_space.sample()
action = sample*0.0
obs,rewards,done,_ =env.step(action)
~~~~~~~